manifests/containerruntimeconfig.crd.yaml

```diff
@@ -14,9 +14,144 @@ spec:
     shortNames:
     - ctrcfg
   scope: Cluster
+  preserveUnknownFields: false
   subresources:
     status: {}
   versions:
-    - name: v1
-      served: true
-      storage: true
+  - name: v1
+    served: true
+    storage: true
+  "validation":
+    "openAPIV3Schema":
```

Comment:
Any idea why it added quotes around these?

Reply:
I tested and that doesn't seem to make a difference either way. The generator generates them as such. See for example crds in api: https://github.com/openshift/api/tree/master/config/v1